metadata.json

```
@@ -7,17 +7,17 @@
     "requires": {
       "~=1.0b": [
             "aiida-core>=1.0.0,<2.0.0",
-            "ase",
-            "bokeh",
-            "fileupload",
-            "ipytree",
-            "ipywidgets",
-            "more_itertools",
-            "nglview",
-            "numpy",
-            "optimade-client",
-            "pandas",
-            "scikit-learn"
+            "ase<3.20",
+            "bokeh~=2.0",
+            "ipytree~=0.2",
+            "ipywidgets<8.0.0",
```

I would think that this would be reasonable:
```suggestion
            "ipywidgets~=7.6.0",
```